orbit/cmd/orbit/shell.go

```diff
@@ -76,9 +78,14 @@ var shellCommand = &cli.Command{
 
 		var g run.Group
 
+		extensionPathPostfix := ""
+		if runtime.GOOS == "windows" {
+			extensionPathPostfix = "-" + uuid.New().String()
```

Comment:
Is each new pipe automatically cleared/deleted when `orbit shell` exits?

Reply:
yes, the named pipe server is bound to the osqueryd process's lifecycle. When orbit shell exits, it will force the child osqueryd process to exit, which will cause the handles to the named pipe server to be closed, forcing the name pipe server to be deleted.